en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/oauth2-access-tokens.md

```diff
@@ -1,15 +1,25 @@
 
--   **Signature** - After validating the signature, Choreo Connect checks whether the JWT is issued by a trusted key manager, and JWT has not tampered in the middle. This signature validation is done by using the public certificate of the key manager who issued the JWT. 
--   **Issuer(iss)** - The issuer claim is a mandatory claim when JWT is used as a security token. Choreo Connect validates the **iss** claim present in the JWT against the issuer provided in the **jwtTokenConfig** section of the configuration.
+## Access Token (JWT) Validation
+
+Choreo Connect can accept JWTs issued by a **trusted** Key Manager as a valid access token to invoke the APIs. JWT tokens are self-validated by Choreo Connect without validating it against the Authorization Server (Key Manager) that issued the JWT.
+
+When an access token (JWT) issued by a trusted Key Manager is used, Choreo Connect validates the following attributes/claims of the JWT.
+
+-   **Signature** - After validating the signature, Choreo Connect checks whether the JWT was issued by a trusted Key Manager, and has not been tampered. This signature validation is done by using the public certificate of the Key Manager that issued the JWT. 
+-   **Issuer(iss)** - The issuer claim is a mandatory claim. Choreo Connect validates the **iss** claim present in the JWT against the issuer provided by the control plane or configured in config.toml.
 -   **Subject(sub)** - The subject claim is also a mandatory claim in a token. Choreo Connect uses the value of the **sub** claim as the user of the secured API.
 -   **Expiry time(exp)** - "exp" claim is also a mandatory claim. Choreo Connect validates the validity period of the token using the **exp** claim
 
-**JWT validation configuration**
+## JWT Validation Configuration
+
+When [Control Plane is enabled]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/), configurations for all Key Managers are fetched from API-M. For [Standalone Mode]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway/), trusted Key Managers can be configured in `config.toml`. Refer to [Configuring an External Key Manager]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/configuring-an-external-key-manager/) to learn more.
+
+Following is an example standalone mode configuration for a trusted Key Manager. When Choreo Connect has API-M as the Control Plane, these configurations will be overridden with the Key Manager configurations retrieved from API Manager if the issuers (Key Managers) are identical. Furthermore, if the corresponding Key Manager is removed from the API Manager Admin portal, the token service added from the configuration will be used.
```

Comment:
shall we elaborate/explain this more `if the issuers (Key Managers) are identical`, if possible

Reply:
Can you give an example?

Reply:
I mean reader may find it difficult to understand what you mean by this phrase `if the issuers (Key Managers) are identical`. Like, is it the name under [[enforcer.security.tokenService]]

Reply:
Got the idea. Will update